cfme/fixtures/cli.py

```diff
@@ -261,9 +261,11 @@ def ha_appliances_with_providers(ha_multiple_preupdate_appliances, app_creds):
     return ha_multiple_preupdate_appliances
 
 
-def replicated_appliances_with_providers(multiple_preupdate_appliances):
-    """Returns two database-owning appliances, configures with providers."""
-    appl1, appl2 = multiple_preupdate_appliances
+def replicated_appliances_with_providers(multiple_appliances):
+    """ Accpets two unconfigured appliance at least.
+    Configures them with different region and sets the Global/Remote kind of
+    replication on them. Configures first appliance with two providers. """
+    appl1, appl2 = multiple_appliances
```

Comment:
One thing that Tasos did in #9829, was to explicitly name these as the global and remote appliances. I think this helps with readability, but I wouldn't consider it a requirement as it creates a lot of boring busy work.

Reply:
This helps quite a lot and I am willing to do that, but perhaps not in this PR. It would also be good because sometimes the appliances are zero-base-indexed and somethines not and this is quite confusing.